data/locale/German.json

```diff
@@ -1,186 +1,189 @@
-    "Possible USB failure. Check your drives power source.": "Möglicher USB Fehler. Prüfe deine Stromversorgung.",
-    "Files might be corrupt": "Dateien könnten fehlerhaft sein",
+    "Install the main game to the same storage medium first": "Installier zuerst das Hauptspiel auf demselben Datenträger",
+    "Also make sure there is no error with the USB drive": "Vergewisser dich auch, dass kein Fehler mit dem USB-Laufwerk vorliegt",
+    "Possible USB error": "Möglicherweise ein USB-Fehler",
+    "Possible missing or bad title.tik file": "Möglicherweise fehlende/fehlerhafte title.tik-Datei",
+    "Possible incorrect console for DLC title.tik file": "Möglicherweise falsche Konsole für die title.tik-DLC-Datei",
+    "Not enough free space on target device": "Nicht genügend Speicherplatz auf dem ausgewählten Datenträger verfügbar",
+    "Files might be corrupt or bad storage medium.\nTry redownloading files or reformat/replace target device": "Möglicherweise sind die Dateien oder das Speichermedium beschädigt.\nVersuch, die Dateien erneut herunterzuladen oder formatiere/ersetze den Datenträger",
+    "Possible USB failure. Check your drives power source.": "Möglicherweise ein USB-Fehler. Überprüf die Stromversorgung deines Laufwerks.",
+    "Files might be corrupt": "Dateien könnten beschädigt sein",
     "Unknown Error": "Unbekannter Fehler",
 
     "Network error": "Netzwerkfehler",
-    "check the network settings and try again": "prüfe die Netzwerkeinstellungen und versuche es nochmal",
-    "SSL error": "SSL Fehler",
-    "check your Wii Us date and time settings": "Prüfe die Datums- und Zeiteinstellungen deiner Wii U",
+    "check the network settings and try again": "Überprüf die Netzwerkeinstellungen und versuch es erneut",
+    "SSL error": "SSL-Fehler",
+    "check your Wii Us date and time settings": "Überprüf die Datums- und Zeiteinstellungen deiner Wii U",
     "Next try in _ seconds.": "Nächster Versuch in _ Sekunden.",
-    "Couldn't resolve hostname": "Konnte Host nicht auflöaen",
-    "Couldn't connect to server": "Konnte nicht zu Server verbinden",
-    "Operation timed out": "Zeit abgelaufen",
-    "The server didn't return any data": "Der Server hat keinerlei Daten zurück geschickt",
-    "I/O error": "I/O Fehler",
+    "Couldn't resolve hostname": "Hostname konnte nicht aufgelöst werden",
+    "Couldn't connect to server": "Verbindung zum Server konnte nicht hergestellt werden",
+    "Operation timed out": "Zeitüberschreitung",
+    "The server didn't return any data": "Der Server hat keine Daten zurückgegeben",
+    "I/O error": "I/O-Fehler",
     "Verification failed": "Verifikation fehlgeschlagen",
-    "Handshake failed": "Handschlag fehlgeschlagen",
+    "Handshake failed": "Handshake fehlgeschlagen",
```

Comment:
Now I'm very interested in the reason you changed other english words into german but here you did the opposite. Not that I think this is wrong, just curious. ;)

Reply:
The proper translation really is "Handschlag" (example: https://de.wikipedia.org/wiki/Drei-Wege-Handschlag ). Anyway, as said what you did was not wrong, was just curious.